config/vufind/permissions.ini

```diff
@@ -59,6 +61,7 @@
 ; ipRegEx = "/1\.2\.3\.4|1\.2\.3\.5/"
 ; ipRange[] = "1.2.3.4"
 ; ipRange[] = "1.2.3.7-1.2.5.254"
+; cookie = "custom cookie value"
```

Comment:
Maybe this example should be something like "VUFIND_CUSTOM_COOKIE_NAME" so it's more realistic.

Reply:
thanks @demiankatz . I've added that in my latest follow-up commit